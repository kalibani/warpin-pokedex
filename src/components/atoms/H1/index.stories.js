// Stories H1 Component
// --------------------------------------------------------
import React, { Fragment } from 'react';
import { storiesOf } from '@storybook/react';
import H1 from './index';
import Notes from './readme.md';

storiesOf('Atoms', module).add('H1', () => (
  <H1 />
), {
  info: { inline: true, header: false, text: Notes }
});
