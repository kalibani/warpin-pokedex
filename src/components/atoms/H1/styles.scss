/* ==========================================================================
Styles for H1 component
========================================================================== */

// import variables
@import "../../../assets/styles/variables";

// Start styles H1 component
.a-h1 {
  font-size: 40px;
  font-weight: 700;
  color: $white;
  margin: 0;
}
