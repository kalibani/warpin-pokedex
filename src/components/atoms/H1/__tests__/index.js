// Test H1 Component
// --------------------------------------------------------

import React from 'react';
import H1 from '../../index';
import renderer from 'react-test-renderer';

test('H1 is match the snapshot', () => {
const component = renderer.create(
H1,
);
let tree = component.toJSON();
expect(tree).toMatchSnapshot();
});
