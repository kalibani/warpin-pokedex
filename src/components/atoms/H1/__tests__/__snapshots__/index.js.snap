// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 is match the snapshot 1`] = `null`;
